ios-drag-drop.js

```diff
@@ -345,6 +337,35 @@ function _exposeIosHtml5DragDropShim(config) {
     } while((el = el.parentNode) && el !== doc.body);
   }
 
+  function elementIsDraggable(el){
+    // if an element is not draggable either explicitly or implicitly we can exit immediately
+    if(!el.draggable === true) return false;
```

Comment:
The 'el.draggable === true' was there before, and I wasn't sure if we were doing a specific boolean comparison for a good reason.  If el.draggable will always be either true or false then I agree with your refactor suggestion.